aura/lib/Aura/Dependencies.hs

```diff
@@ -125,8 +132,8 @@ sortInstall m = case cycles depGraph of
         singles   = overlays $ map vertex elems
 
 cycles :: Ord a => AdjacencyMap a -> [AdjacencyMap a]
-cycles x = [ induce (`S.member` c) x | c <- cs ]
-  where cs = filter (\c -> S.size c > 1) $ vertexList (scc x)
+cycles x = [ induce (`NAM.hasVertex` c) x | c <- cs ]
+  where cs = filter (\c -> NAM.vertexCount c > 1) . vertexList $ scc x
```

Comment:
My hope is that this is a one-to-one change. @snowleopard ? :)

Reply:
Yes, I think your change is correct, but perhaps it would be better switch to using [`isAcyclic`](http://hackage.haskell.org/package/algebraic-graphs-0.3/docs/Algebra-Graph-ToGraph.html#v:isAcyclic):

```haskell
cycles x = [ induce (`NAM.hasVertex` c) x | c <- cs ]
  where cs = filter (not . isAcyclic) . vertexList $ scc x
```

This seems clearer, and also correctly handles cycles comprising a single vertex (although I doubt your application can actually have such cycles).

Reply:
Wait, I think this can now be simplified!

```
cycles = map toAdjacencyMap . filter (not . isAcyclic) . vertexList . scc
```